signer/xpi/x509.go

```diff
@@ -56,30 +56,45 @@ func (s *XPISigner) monitorRsaCacheSize() {
 	}
 }
 
-// retrieve a key from the cache or generate one if it takes too long
-// or if the size is wrong
+var currentRsaKeyAge time.Time
+var currentRsaKeyUsage int
+var currentRsaKey *rsa.PrivateKey
```

Comment:
nit: this can be one var block or maybe we want to throw the age, usage, key, and chan in an RSAKeyCache struct?